src/inc/til/rect.h

```diff
@@ -294,9 +293,9 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
         }
 
         // - = subtract
-        constexpr some<rect, 4> operator-(const rect& other) const
+        small_vector<rect, 4> operator-(const rect& other) const

```

Comment:
so, the biggest issue with small_vector is that it no longer guarantees _only up to_ N items. that was somewhat the real contract of `some`

Reply:
I believe in the cases where `some` was used so far, this difference didn't matter.